apps/site/src/app/login/components/LoginForm.tsx

```diff
@@ -1,30 +1,35 @@
+import Button from "@/lib/components/Button/Button";
 import ValidatingForm from "@/lib/components/ValidatingForm/ValidatingForm";
 
-const EMAIL_REGEX = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+const EMAIL_REGEX = /^\w+([\.\-]?\w+)*@\w+([\.\-]?\w+)*(\.\w{2,3})+$/;
```

Comment:
Note escaping `.` and `-` within the character class shouldn't be required but I understand your browser seemed to be enabling strict unicode mode or something.